### ISSUE TITLE
Add possibility to restore Bitmap from a file. 

### ISSUE THREAD

**Issue by opankiv** [RiccardoMoro/FreeDrawView#20]
Read Bitmap from a file and load it to FreeDrawView.

**Comment by RiccardoMoro** [comment]
For now it supports state save and restore only with the following methods
`FreeDrawSerializableState  getCurrentViewStateAsSerializable()`
and 
`restoreStateFromSerializable(FreeDrawSerializableState state)`

**Comment by opankiv** [comment]
Yes. But it will be useful to have an ability to read Bitmap data from an external file (maybe previously saved locally) and load it to FreeDrawView for further editing.

**Comment by wlmirand** [comment]
Agreed. Used the Library to save a signature into a Bitmap file, but needed to load it from the file.
